### PR TITLE
Update colors.md

### PR DESCRIPTION
**Pull request by Datseris**
Closes https://github.com/JuliaPlots/Makie.jl/issues/1471

I'll be totally honest: I have no clue whatsoever how docs work and what this `{{colorschemes}}` business does.

### CODE REVIEW

File: docs/documentation/colors.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # Colors
 
-{{colorschemes}}
\ No newline at end of file
+## Named colors
+Named colors in Makie.jl (e.g., `:blue`) are parsed using [Colors.jl](https://juliagraphics.github.io/Colors.jl/stable/constructionandconversion/#Color-Parsing) and thus have a large array of possibilities under CSS specifications. You can find a plotted table of all possible names in this page: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#color_keywords
```
Comment (jkrumbiegel) [inline docs/documentation/colors.md:4]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#color_keywords

that should also be a link I think

Comment (Datseris) [reply docs/documentation/colors.md:4]:
```suggestion
Named colors in Makie.jl (e.g., `:blue`) are parsed using [Colors.jl](https://juliagraphics.github.io/Colors.jl/stable/constructionandconversion/#Color-Parsing) and thus have a large array of possibilities under CSS specifications. You can find a plotted table of all possible names [in this page](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#color_keywords).
```

**Comment by jkrumbiegel** [comment]
thanks!

**Comment by lazarusA** [comment]
I now that is closed and merged, but I always prefer to use this link for named colors:
http://juliagraphics.github.io/Colors.jl/stable/namedcolors/

**Pull request merged**